android/tutorials/tutorial3/README.md

```diff
@@ -275,12 +275,29 @@ Building the project at this point, Dagger should give you:
 
 ## Bubbling up dependencies
 
-This will be super easy, if you got this far. The only difference is that if we cannot provide a dependency directly, we also add it to the `Dependency` interface of the parent (and then to its parent and so on until we can actually provide id).
+This will be super easy, if you got this far. The only difference is that if we cannot provide a dependency directly. We also add it to the `Dependency` interface of our rib and to the 'Dependency' interfaces of all parent ribs until this rib can actually provide id.
```

Comment:
Your change here does not make sense and changes the content for the worse unfortunately.

"If we cannot provide --> then we also add it" semantically belong together.
Also doesn't make sense to replace "parent" with "our rib" since it's really the parent the text needs to refer to.

Reply:
made the sentence sematically correct, changed "our rib" back to "parent"